### ISSUE TITLE
Issue

### ISSUE THREAD

**Issue by ekowpo** [SOEN341-Group1/TemporaryName#46]
Do we add a document to the event to a calendar on creation or we should have a different view to add documents to an event so as to accommodate multiple files addition and description for each event?

**Comment by g-harel** [comment]
I would make it on creation or on edit

**Comment by ekowpo** [comment]
So technical, 

1. create a calendar,
2. add an event.
3. In the add event, you will have file upload & description.

How many files per description?

**Comment by g-harel** [comment]
Let's start with no file description and see if we want to add that feature in the future. We are going for minimal viable product

**Comment by ekowpo** [comment]
So basically we have only assignment right

**Comment by g-harel** [comment]
Not sure what you mean

**Comment by ekowpo** [comment]
to add a pdf file(assignment) to the event.

**Comment by g-harel** [comment]
Yeah I think it would be better to keep things simpler until we are farther along
